config/server-config.yaml.example

```diff
@@ -65,6 +65,9 @@ webhook-config:
   external_webhook_url: "https://example.com/api/v1/webhook/github"
   external_ping_url: "https://example.com/api/v1/health"
   webhook_secret: "your-password"
+#  previous_webhook_secrets:
+#   - "under-migration"
+#   - "should-be-rotated"
```

Comment:
Should these be strings in configuration, or references to either files on disk or environment variables?

What do you think of having a single file with a list of accepted webhook secrets?

Reply:
Good point (both you and @JAORMX). I'll change the option to be akin `auth.token_key` to reference a file

Reply:
done!